src/components/Select/index.tsx

```diff
@@ -78,10 +29,9 @@ const Select: FunctionComponent<SelectProps> = forwardRef(
           {...props}
           ref={ref}
           theme={theme}
-          aria-required={required}
+          required={required}
```

Comment:
probably don't need this as required probably exists already in SelectHTMLAttributes<HTMLSelectElement>